code/api/src/seeders/3-crates.js

```diff
@@ -1,43 +1,44 @@
 'use strict';
+import moment from 'moment'
 
 module.exports = {
   up: (queryInterface, Sequelize) => {
     return queryInterface.bulkInsert('crates', [
       {
         name: 'Clothes for Men',
         description: 'A monthly supply of trendy clothes for men.',
-        createdAt: new Date(),
-        updatedAt: new Date()
+        createdAt: moment(),
+        updatedAt: moment()
       },
       {
         name: 'Clothes for Women',
         description: 'A monthly supply of trendy clothes for women.',
-        createdAt: new Date(),
-        updatedAt: new Date()
+        createdAt: moment(),
+        updatedAt: moment()
       },
       {
         name: 'Accessories for Men',
         description: 'A monthly supply of trendy accessories for men',
-        createdAt: new Date(),
-        updatedAt: new Date()
+        createdAt: moment(),
+        updatedAt: moment()
       },
       {
         name: 'Accessories for Women',
         description: 'A monthly supply of trendy accessories for women',
-        createdAt: new Date(),
-        updatedAt: new Date()
+        createdAt: moment(),
+        updatedAt: moment()
       },
       {
         name: 'Clothes and Accessories for Men',
         description: 'A monthly supply of trendy clothes and accessories for men',
-        createdAt: new Date(),
-        updatedAt: new Date()
+        createdAt: moment(),
+        updatedAt: moment()
       },
       {
         name: 'Clothes and Accessories for Women',
         description: 'A monthly supply of trendy clothes and accessories for women',
-        createdAt: new Date(),
-        updatedAt: new Date()
+        createdAt: moment(),
+        updatedAt: moment()
       }
```

Comment:
Updated all these to moment() form new Date() so that it was simpler to make calculations for estimated date of arrival of subscribed crates. Currently, it will display on page if you click on "subscriptions" and then go to "profile".

Reply:
Thanks for explaining this so clearly, Carlos!